### PR TITLE
Update compliance-manager-improvement-actions.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated get details information to tell which actions are tested automatically and which cannot be tested.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9520

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/compliance-manager-improvement-actions.md [hunk 1]
```diff
@@ -149,7 +149,7 @@ Compliance Manager provides you options for how to test improvement actions. In
 Improvement actions set for manual testing are actions which you manually test and implement. You set the necessary implementation and test status states, and upload any evidence files on the **Documents** tab. For some actions, this is the only available method for testing improvement actions.
 
 #### Automatic testing source
-Certain improvement actions can be automatically tested by Compliance Manager. [Get details](compliance-manager-improvement-actions.md#update-testing-source) on which improvement actions can and can't be tested automatically.
+Certain improvement actions can be automatically tested by Compliance Manager. [Get details](/compliance/compliance-manager-setup?view=o365-worldwide#set-up-automated-testing) on which improvement actions can and can't be tested automatically.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/compliance-manager-improvement-actions.md]:
```suggestion
Certain improvement actions can be automatically tested by Compliance Manager. [Get details](/compliance/compliance-manager-setup?view=o365-worldwide#set-up-automated-testing) about which improvement actions can and can't be tested automatically.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chvukosw Please merge this PR. Thanks

**Comment by chvukosw** [comment]
Link not set correctly. Closing this PR, fixing in a separate one being merged today.

**Pull request closed without merging**